win32_event_log/datadog_checks/win32_event_log/win32_event_log.py

```diff
@@ -175,7 +180,7 @@ def _dt_to_wmi(self, dt):
 
 class LogEvent(object):
     def __init__(self, ev, log, hostname, tags, notify_list, tag_event_id, event_format, event_priority):
-        self.event = ev
+        self.event = LogEvent._normalize_event(deepcopy(ev))
```

Comment:
We could do a shallow copy, no?

Reply:
You can use self for static methods too.

Reply:
👍 

Yes, it's probably enough for now to do a copy.

Used self instead.